src/Mailgun/Api/Suppressions.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Api;
+
+use Http\Client\HttpClient;
+use Mailgun\Deserializer\ResponseDeserializer;
+use Mailgun\RequestBuilder;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class Suppressions
```

Comment:
@Nyholm the `Suppressions` class is just another API gateway, like the top-level Mailgun class.